.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: rust
+script:
+  - rev=$(git rev-parse --short HEAD)
+  - (cargo install mdbook --vers 0.0.26 --force || true)
```

Comment:
Why `|| true`? If this step fails, wouldn't the CI script fail on the next line, since `mdbook` might not be available?

Reply:
We copied this over from the rust RFC repo. Unsure about this line's significance

Reply:
Thanks 👌